library/src/main/java/com/firebase/ui/FirebaseLoginDialog.java

```diff
@@ -0,0 +1,73 @@
+package com.firebase.ui;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.hardware.camera2.params.Face;
```

Comment:
On a side note, ctrl + alt + O should optimize imports and remove unused ones on any given file. Might be worth running a few of there, or just use the Android Studio "Optimize imports" option